### PR TITLE
[euphoria-beam-io] Added beam wrappers for euphoria Sinks and Sources

### PR DESCRIPTION
**Pull request by mareksimunek**

### CODE REVIEW

File: euphoria-beam-io/src/main/java/cz/seznam/euphoria/beam/io/EuphoriaIO.java [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    }
+
+  }
+
+  /**
+   * Write transform for euphoria sinks.
+   *
+   * @param <T> type of data to write
+   */
+  public static class Write<T> extends PTransform<PCollection<T>, PDone> {
```
Comment (dmvk) [inline euphoria-beam-io/src/main/java/cz/seznam/euphoria/beam/io/EuphoriaIO.java]: we should get rid of Write transform completely

File: euphoria-beam-io/src/main/java/cz/seznam/euphoria/beam/io/EuphoriaIO.java [hunk 2]
```diff
@@ -0,0 +1,234 @@
+package cz.seznam.euphoria.beam.io;
```
Comment (dmvk) [inline euphoria-beam-io/src/main/java/cz/seznam/euphoria/beam/io/EuphoriaIO.java]: licsense

Comment (mareksimunek) [reply euphoria-beam-io/src/main/java/cz/seznam/euphoria/beam/io/EuphoriaIO.java]: added

**Comment by mareksimunek** [comment]
@dmvk EuphoriIO.writer deleted, added lincenses.

File: gradle/versions.gradle [hunk 3]
```diff
@@ -17,7 +17,7 @@ ext {
   mockitoVersion = '2.10.0'
 
   // ~ benchmark dependencies
-  beamVersion = '0.6.0'
+  beamVersion = '2.7.0'
```
Comment (dmvk) [inline gradle/versions.gradle]: Should the version be 2.8.0?

Comment (mareksimunek) [reply gradle/versions.gradle]: If 2.8.0 would be released then yes :), but its not yet

**Pull request merged**